django_mongodb/query.py

```diff
@@ -102,7 +107,17 @@ def get_cursor(self):
                 # If name != column, then this is an annotatation referencing
                 # another column.
                 fields[name] = 1 if name == column else f"${column}"
+
+        # Add the subquery results if fields is defined.
+        if fields:
+            for alias in self.query.alias_map:
+                if self.query.alias_refcount[alias] and self.collection_name != alias:
+                    fields[alias] = 1
+
         pipeline = []
+        if self.mongo_lookups:
+            lookups = self.mongo_lookups
```

Comment:
the `lookups` variable seems unnecessary (possibly a debugging or refactoring artifact)